importexport/CMakeLists.txt

```diff
@@ -58,9 +57,14 @@ add_library (
       ${BWW_SRC}
       )
 
+add_subdirectory(../thirdparty/beatroot beatroot) #for midiimport
+add_subdirectory(../thirdparty/rtf2html rtf2html) #for capella
+
 target_link_libraries(${MODULE}
     ${QT_LIBRARIES}
-    midi        # for midi import
+    rtf2html    #for capella
+    midi        #for midiimport
+    beatroot    #for midiimport
```

Comment:
nitpicks, but I'd prefer a space after the `#`s, so it doesn't look like a preprocessor command ;-)

Reply:
fixed :)

Reply:
thanks!